### PR TITLE
fixes searchbar jumps to newline

### PR DESCRIPTION
**Pull request by lory87**
**This pull request fixes the following issues**:

- fixed on mobile and tablet view that searchbar button don't jumps to newline

### CODE REVIEW

**Comment by Flight** [comment]
ДПЕЬ

File: djangocms_admin_style/sass/components/_mobile.scss [hunk 1]
```diff
@@ -65,6 +65,9 @@ body.login {
     float: none !important;
     width: 100% !important;
 }
+.filebrowser #content-top-bar .object-tools {
```
Comment (ruslanmalogulko) [inline djangocms_admin_style/sass/components/_mobile.scss]: do you need such specific selector?

**Comment by dhorelik** [comment]
LGTM, as long as the searchbar looks good

File: djangocms_admin_style/sass/components/_mobile.scss (hunk 1 above)
Comment (lory87) [inline djangocms_admin_style/sass/components/_mobile.scss]: unfortunately yes

**Pull request merged**